doc/rules/php_tag/inline_tag.rst

```diff
@@ -0,0 +1,171 @@
+===================
+Rule ``inline_tag``
+===================
+
+Changes spaces and semicolons in inline PHP tags.
+
+Configuration
+-------------
+
+``space_before``
+~~~~~~~~~~~~~~~~
+
+The desired number of spaces at the beginning of the tag.
+
+Allowed values: ``'keep'``, ``'minimum'``, ``'one'``
+
+Default value: ``'one'``
+
+``space_after``
+~~~~~~~~~~~~~~~
+
+The desired number of spaces at the end of the tag.
+
+Allowed values: ``'keep'``, ``'minimum'``, ``'one'``
+
+Default value: ``'one'``
+
+``semicolon``
+~~~~~~~~~~~~~
+
+Whether there should be a semi-colon at the end.
+
+Allowed types: ``bool``, ``null``
+
+Default value: ``false``
```

Comment:
I would strongly prefer `true` as a default, code is more readable then and more proof to errors.

Reply:
Do you find more readable

```php
<?= $a; ?>
```

than

```php
<?= $a ?>
```

?

Reply:
@mvorisek also, why "more proof to errors"?

Reply:
https://3v4l.org/5JfNF more readable when there is more than one statement

Reply:
IMHO semicolons in inline tags are very rare. BTW if changing this value will speedup the approval of this fixer I'd be more than happy to change jt

Reply:
I would keep default without semicolons. To be honest, not sure whether we actually need it configurable

Reply:
> To be honest, not sure whether we actually need it configurable

Someone may prefer to never have trailing semicolons (like me), someone else may prefer to always have them (like @mvorisek)